README.md

```diff
@@ -17,6 +17,9 @@ We have a small collection of algorithms, one for every day of the month. Scroll
 - [**December 8 - Singular-Plural**](#december-8---singular-plural)
 - [**December 9 - Is this an URL?**](#december-9---is-this-an-url)
 - [**December 10 - Find the Determinant**](#december-10---find-the-determinant)
+- [**December 11 - Printing matrix in spiral form**](#december-11---printing-matrix-in-spiral-form)
```

Comment:
Title Case the index.

December 7 - Are They Approximately Equal?
December 11 - Printing a Matrix in Spiral Form
December 12 - Reversing a Singly Linked List
December 13 - Lexicographical Arrangement